backend/src/main/java/gov/cdc/usds/simplereport/service/ApiUserService.java

```diff
@@ -376,6 +376,21 @@ public UserInfo resendActivationEmail(UUID userId) {
     return new UserInfo(apiUser, Optional.of(orgRoles), false);
   }
 
+  /**
+   * Clear a user's roles and facilities - intended to be used on site admin users only,
+   * non-site-admin users will be in a misconfigured state without roles and facilities
+   *
+   * @param username
+   * @return ApiUser
+   */
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public ApiUser clearUserRolesAndFacilities(String username) {
+    ApiUser foundUser =
+        _apiUserRepo.findByLoginEmail(username).orElseThrow(NonexistentUserException::new);
+    foundUser.clearRolesAndFacilities();
```

Comment:
worth considering for a followup PR: could we stick a check for whether the found user is a site admin? Think this would entail extending our site admin validation code since from what I can tell, it grabs that information from the currently-logged in user auth properties, but think that 

1) doing an explicit check is the safest way to make sure we don't end up in a unsupported state
2) Having a "is this email in the site admin group" is a generally useful method to have.

Since all this is behind a feature flag / this mutation is needed to clean up a testing env, no need to address here, but to consider as a follow up

Reply:
Created this issue to keep track of this effort: https://github.com/CDCgov/prime-simplereport/issues/8001 
Thank you, @fzhao99! 😸